### DISCUSSION TITLE
Linking 2 collections

### DISCUSSION THREAD

**Discussion by dtau00** [appy-one/acebase#77] [Q&A] [answered]
Im trying to find the best way to link 2 collections using a live proxy.  So lets say I have a collection of Groups and GroupMembers.  I want to create a live proxy of groups and all its members.  Obviously a group can have many group members (linked via group_id).  Basic db stuff.  How would i load a group, while also loading a filtered collection of group members in the group, and attach it to a proxy?  And along the same lines, is it possible to query/filter when creating a proxy?

1.  I tried this syntax i saw somewhere but didnt work
                 this.proxy = await db()
                  .ref(this.dbPath)
                  .query()
                  .filter(queryFilters|'')
                  .proxy();
2.  Another member suggested this might be possible with an index.  The documenation seems to be suggesting instead of having 2 collections, i make group memebers chilren of each group.  Then i would create an index pull it all together?

**Comment by appy-one** [comment, marked as answer]
A live data proxy is the in-memory representation of what is in your database. It is automatically updated with remote changes and pushes local changes back to the database. You cannot query it, filter what is loaded or combine it with other data.
